### ISSUE TITLE
Unable to make custom update on my playbook (sudo rights)

### ISSUE THREAD

**Issue by fatimazahrae2104** [ansible/awx#9243]
Hi,

I Installed AWX on my Openshift Cluster, and it works fine with the project suggested on https://github.com/ansible/ansible-tower-samples, but when I tried to make a simple modification on the playbook as mentioned bellow, I got this message error.

> {
  "module_stdout": "",
  "module_stderr": "Password: su: Authentication failure\n",
  "msg": "MODULE FAILURE\nSee stdout/stderr for the exact error",
  "rc": 1,
  "_ansible_no_log": false,
  "changed": false
}

- My Playbook after custom update for test.

![image](https://user-images.githubusercontent.com/36632123/107043306-1b587e00-67c3-11eb-9eab-a4e8e96ec8e7.png)

Can Anyone help me about this?!
Thanks in advance.

**Comment by wenottingham** [comment]
This issue tracker is for tracking feature enhancements and bugs to AWX itself.

You look like you're trying to install RPMs inside the task container where the playbook is running - why would you want to do that?
In any case, if you need help troubleshooting an AWX install, try our mailing list or IRC channel:

http://webchat.freenode.net/?channels=ansible-awx
https://groups.google.com/forum/#!forum/awx-project

**Comment by fatimazahrae2104** [comment]
Hi @wenottingham, I already successfully installed AWX, but Actually my problem is in the usage of AWX on Openshift cluster not in the installation.

Normally the AWX Interface comes by default with a template created "Demo Job Template" which uses a project from https://github.com/ansible/ansible-tower-samples, this project contain a very basic playbook which runs successfully without any problem, but for me I need to add more custom tasks to the playbook, as for example installing packages and services... in managed hosts, and to do this I tried before to add a host IP Address (which exists in the same namespace where I installed AWX) in the Inventory, but I got ssh connection problem:

`{
  "unreachable": true,

  "msg": "Failed to connect to the host via ssh: ssh: connect to host 172.21.245.51 port 22: Connection timed out",

  "changed": false
}`

And even if I use only localhost without adding any other Host in the inventory, I cannot add some tasks, like the task I shared in the comment above, so I get the privilege rights problem, I need to connect as a superuser, I already created a superuser by 
  `awx-manage createsuperuser` command but I think it's not what I need.

are the channels you mentioned to me in your comment, for these types of problems?
